spread.yaml

```diff
@@ -38,48 +70,57 @@ exclude:
 path: /home/test/src/github.com/snapcore/spread
 
 suites:
-    tests/: 
+    tests/main/:
         summary: Integration tests
+        backends: [google-nested, qemu]
+        prepare: |
+            DEBS="git qemu-kvm lxd tree"
```

Comment:
This is duplicated with what's installed in github actions. You can probably just "apt-get update; apt-get install $DEBS" instead of using the loop below.